regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -74,9 +74,7 @@ var CommentReviewView = Backbone.View.extend({
   serialize: function() {
     // TODO(vrajmohan) Add other regs.gov fields
     return {
-        assembled_comment:  {
-            sections: comments.toJSON({docId: this.docId})
-        }
+      assembled_comment: comments.toJSON({docId: this.docId})
```

Comment:
There seems to be some miscommunication here. I assumed that the full comment payload, when the other fields were added per L75 would be something like:
```
{"assembled_comment": {
    "sections" : [
    {"id": "A1", "comment": "A simple comment", "files": [{"name": "doc0.pdf", "key": "doc0.pdf"}]}
    {"id": "A5", "comment": "Another comment", "files": [{"name": "doc1.pdf", "key": "doc2.pdf"}, {"name": "doc1.pdf", "key": "doc2.pdf"}]}
]},
"submitter_rep": "The Advocate"}
```
As I mentioned to @cmc333333 in another PR comment, we could lose the `sections` level as it adds no information.

For the existing code, the only change we would need would be to either handle `assembled_comment.sections` in the template, or pass `sections` to the `render` call.